assemblies/features/base/src/main/filtered-resources/resources/bin/client

```diff
@@ -125,16 +125,16 @@ init() {
 
 run() {
     convertPaths
-    exec "${JAVA}" ${JAVA_OPTS} \
-        -Dkaraf.instances="${KARAF_HOME}/instances" \
-        -Dkaraf.home="${KARAF_HOME}" \
-        -Dkaraf.base="${KARAF_BASE}" \
-        -Dkaraf.etc="${KARAF_ETC}" \
-        -Dkaraf.log="${KARAF_LOG}" \
-        -Djava.io.tmpdir="${KARAF_DATA}/tmp" \
-        -Djava.util.logging.config.file="${KARAF_BASE}/etc/java.util.logging.properties" \
-        ${KARAF_OPTS} ${OPTS} \
-        -classpath "${CLASSPATH}" \
+    exec "${JAVA}" ${JAVA_OPTS} /
```

Comment:
I guess it's unexpected right ?

Reply:
Yes, too many replaces of backslashes. :-)